zerver/finders.py

```diff
@@ -11,7 +11,7 @@ class ExcludeUnminifiedMixin(object):
     def list(self, ignore_patterns):
         # We can't use ignore_patterns because the patterns are
         # applied to just the file part, not the entire path
-        excluded = '^(js|styles|templates)/'
+        excluded = '^(js|templates)/'
```

Comment:
Yes that could be done. I think that's what we're doing for source-maps right now. I'll modify tools/build-release-tarball to do that.